### PR TITLE
fix(*): reformat code for lint

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

fix(*): reformat code for lint

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hezhizhen @zxq-bit 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/release-cli/get.go [hunk 1]
```diff
@@ -105,7 +105,11 @@ func runGet(cmd *cobra.Command, args []string) {
 		fmt.Println("Template:")
 		buf := bytes.NewBuffer(nil)
 		encoder := base64.NewEncoder(base64.StdEncoding, buf)
-		encoder.Write(r.Spec.Template)
+		_, err := encoder.Write(r.Spec.Template)
+		if err != nil {
+			fmt.Println("")
```
Comment (zxq-bit) [inline cmd/release-cli/get.go]: 是不是应该打印错误而非空串？

File: pkg/render/render.go [hunk 2]
```diff
@@ -139,14 +139,15 @@ func (r *render) isHook(resource string) (bool, error) {
 	return ok, nil
 }
 
-func (r *render) renderConfig(options *RenderOptions) (string, error) {
+func (r *render) renderConfig(options *Options) (string, error) {
 	if options.Suspend == nil || (options.Suspend != nil && !*options.Suspend) {
 		return options.Config, nil
 	}
 
+	var err error
```
Comment (zxq-bit) [inline pkg/render/render.go]: 这个放在 cb 里面也无妨的样子，反正执行完就被 ArrayEach 结果覆盖掉了
别处直接用是因为这样改的少，这里就不一定非要放在外面了

Comment (whalecold) [reply pkg/render/render.go]: 放在里面 lint 会报错，所以就在外面了。

File: cmd/release-cli/get.go (hunk 1 above)
Comment (whalecold) [reply cmd/release-cli/get.go]: done.

File: pkg/render/render.go (hunk 2 above)
Comment (zxq-bit) [reply pkg/render/render.go]: ok

**Comment by zxq-bit** [comment]
LGTM
/cc @hezhizhen

**Pull request merged**